students/JerryH/Lesson04/trigrams.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import string
+import random
+from collections import defaultdict
+
+
+def get_text():
+    file = open("sherlock.txt", "r")
+    text = file.read()
+    file.close()
```

Comment:
keep an eye on var names that overwrite built-in objects, you can see git highlights them in orange